editors/tea/Portfile

```diff
@@ -61,8 +73,10 @@ variant qt5 conflicts qt4 description {Use Qt 5} {}
 
 if {[variant_isset qt4]} {
     PortGroup           qmake 1.0
+
 }
 
 if {[variant_isset qt5]} {
     PortGroup           qmake5 1.0
+
```

Comment:
Why the extra carriage return?

Reply:
I do not know specific rationale, but lint wants it.